### ISSUE TITLE
Data missing from ooni-explorer search results

### ISSUE THREAD

**Issue by anadahz** [ooni/pipeline#65]
Some data are missing from the [ooni-explorer](https://explorer.ooni.torproject.org) search results.
For instance the results from the following web_connectivity test report file are missing:
https://measurements.ooni.torproject.org/files/download/20170516T011345Z-CY-AS197792-web_connectivity-20170516T011346Z_AS197792_GRkCdSXa2dhdmp6FKPA7NBftModMSjHrpcOPPWhoixwg3dgUlw-0.2.0-probe.json

**Comment by darkk** [comment]
This report is stored in S3 (obviously).
It's also properly transferred to datacollector box.
The report file is not imported into DB because of postgres being strict about `\u0000` characters in the data. It's [recently introduced](https://github.com/TheTorProject/ooni-probe/commit/582f85501a072dfd8f2f6ebbdb28dd615a8db325#diff-39d69468c9e012e0b520b5ae35fe2074L11) "regression", but the commit should not be revered.
IMHO, it's not worth fixing it in luigi-based pipeline as it's death is currently scheduled to Jul 22 (given data growth rate): as far as I see, partial re-processing is quite painful with luigid and I'd rather spend time delivering stable ticking pipeline-16.10 :)

<details><summary>Backtrace</summary>

```
2017-05-17 06:25:32,417 - luigi-interface - ERROR - [pid 1174] Worker Worker(salt=433161014, workers=20, host=chameleon, username=soli, pid=26600) failed    InsertMeasurementsIntoPostgres(report_path=/data/ooni/private/reports-raw/yaml/2017-05-17/20170516T011345Z-CY-AS197792-web_connectivity-20170516T011346Z_AS197792_GRkCdSXa2dhdmp6FKPA7NBftModMSjHrpcOPPWhoixwg3dgUlw-0.2.0-probe.json)
Traceback (most recent call last):
  File "/home/soli/venv/local/lib/python2.7/site-packages/luigi/worker.py", line 192, in run
    new_deps = self._run_get_new_deps()
  File "/home/soli/venv/local/lib/python2.7/site-packages/luigi/worker.py", line 130, in _run_get_new_deps
    task_gen = self.task.run()
  File "/home/soli/venv/local/lib/python2.7/site-packages/luigi/postgres.py", line 326, in run
    self.copy(cursor, tmp_file)
  File "/home/soli/venv/local/lib/python2.7/site-packages/luigi/postgres.py", line 289, in copy
    cursor.copy_from(file, self.table, null=r'\\N', sep=self.column_separator, columns=column_names)
DataError: unsupported Unicode escape sequence
DETAIL:  \u0000 cannot be converted to text.
CONTEXT:  JSON data, line 1: ...t_ip":null,"exit_name":null}},"response":{"body":...
COPY metrics, line 610, column test_keys: "{"accessible":true,"control":{"tcp_connect":{"162.209.96.185:80":{"status":true,"failure":null}},"ht..."
```
</details>

**Comment by darkk** [comment]
Aforementioned `\0` is fixed since 45d12b1a524874658f7a0a41cb135eb2c6a72571.
And the link works :)
